packages/test-runner-saucelabs/src/createSauceLabsLauncher.ts

```diff
@@ -27,9 +34,14 @@ export function createSauceLabsLauncher(
     finalSauceLabsOptions.region = 'us';
   }
 
-  const finalConnectOptions: SauceConnectOptions = { ...sauceConnectOptions };
-  if (typeof finalConnectOptions.tunnelIdentifier !== 'string') {
-    finalConnectOptions.tunnelIdentifier = `web-test-runner-${nanoid()}`;
+  const finalConnectOptions: SauceConnectOptions & LegacySauceConnectOptions = { ...sauceConnectOptions };
+  if (finalConnectOptions?.tunnelIdentifier) {
+    console.warn('The `tunnelIdentifier` option is deprecated. Use `tunnelName` instead.');
+    finalConnectOptions.tunnelName = finalConnectOptions.tunnelIdentifier;
+    delete finalConnectOptions.tunnelIdentifier;
+  }
```

Comment:
This option was an obvious change and it was easy to provide a migration path for it. Up to you if you want to include it.

Reply:
I'd rather not, such changes are expected in minor version in 0.x.x range, and I think we are just a proxy to SauceLabs libs, so we don't need to do more that they do, if they remove it, we should follow and not make own deprecation policy.

Should you fix this, I'll be happy to merge and release this update, good work!

Reply:
Alright, I removed it 🙂